percolator/README.md

```diff
@@ -29,6 +29,8 @@ This workload allows to overwrite the following parameters with Benchmark 0.8.0+
 * `index_settings`: A list of index settings. Index settings defined elsewhere (e.g. `number_of_replicas`) need to be overridden explicitly.
 * `cluster_health` (default: "green"): The minimum required cluster health.
 * `error_level` (default: "non-fatal"): Available for bulk operations only to specify ignore-response-error-level.
+* `target_throughput` (default: default values for each operation): Number of request per second, `none` for no limit.
```

Comment:
Minor typo: Number of requests

Also, would be clearer to describe what the default value represents,( here and in the other occurrences):
"default: pre-configured default value for each operation"